File: 1 - Iniciante/C99/BEE1131.c
#include <stdio.h>

int main(void){
	int gremio, inter;
	int empates=0, vinter=0, vgremio=0;
	int qtd_grenais=1, novo_grenal;
	
	while (1){
		scanf("%d %d", &inter, &gremio);
		
		if (gremio>inter)	
			vgremio++;
		else if (inter>gremio)
			vinter++;
		else 
			empates++;
		
		printf("Novo grenal (1-sim 2-nao)\n");
		scanf("%d", &novo_grenal);
		
		if (novo_grenal == 1)
			qtd_grenais++;
		else
			break;	
	}
	
	printf("%d grenais\n", qtd_grenais);
	printf("Inter:%d\n", vinter);
	printf("Gremio:%d\n", vgremio);
	printf("Empates:%d\n", empates);
	if (inter>gremio)
		printf("Inter venceu mais\n");
	else if (gremio>inter)
		printf("Gremio venceu mais\n");
	else 
		printf("Nao houve vencedor\n");
	
}

File: 1 - Iniciante/Python 3.11/BEE1131.py
inter = 0
gremio = 0
qtd_grenais = 0
empates = 0
novo_grenal = True
while (novo_grenal):
    x, y = map(int, input().split(" "))
    
    if (x>y):
        inter += 1
    elif (y>x):
        gremio += 1
    else:
        empates += 1

    while (True):
        print("Novo grenal (1-sim 2-nao)")
        opcao = int(input())
        match (opcao):
            case 1:
                novo_grenal = True
                break
            case 2:
                novo_grenal = False
                break
            case _:
                continue

    qtd_grenais += 1

print(f'{qtd_grenais} grenais')
print(f'Inter:{inter}')
print(f'Gremio:{gremio}')
print(f'Empates:{empates}')
if (inter>gremio):
    print("Inter venceu mais")
elif (gremio>inter):
    print("Gremio venceu mais")
else:
    print("Nao houve vencedor")